src/Classes/Entities/EventEntity.php

```diff
@@ -4,6 +4,7 @@
 
 class EventEntity
 {
+    protected $id;
```

Comment:
Why did you add id? The id is generated by the database, not supplied by the user.

Reply:
Yeah I was confused if I should add this or not, seeing that they did in the ApplicantEntity, I did the same. I will remove it.